docarray/document/mixins/plot.py

```diff
@@ -185,15 +185,24 @@ def display_point_cloud_tensor(self) -> None:
         from IPython.display import display
         from hubble.utils.notebook import is_notebook
 
+        colors = np.tile(np.array([0, 0, 0]), (len(self.tensor), 1))
+        for chunk in self.chunks:
+            if (
+                'name' in chunk.tags.keys()
+                and chunk.tags['name'] == 'point_colors'
```

Comment:
this point_colors string should come from some common variable, otherwise is just a magic name

Reply:
yes